phoenix-core/src/main/java/org/apache/phoenix/util/SchemaUtil.java

```diff
@@ -983,14 +983,21 @@ public static boolean hasRowTimestampColumn(PTable table) {
         return SchemaUtil.getTableKey(null, schemaName, MetaDataClient.EMPTY_TABLE);
     }
 
-    public static PName getPhysicalHBaseTableName(PName pName, boolean isNamespaceMapped, PTableType type) {
-        return getPhysicalHBaseTableName(pName.toString(), isNamespaceMapped, type);
+    public static PName getPhysicalHBaseTableName(PName schemaName, PName tableName, boolean isNamespaceMapped) {
```

Comment:
I added "schemaName" as an argument to this method.
I think we need both schemaName and tableName to handle a table like "AAA.BBB" in default schema correctly, because if we have only the table name "AAA.BBB", we can't distinguish between "AAA.BBB" in the default schema and "BBB" in "AAA" schema.

Also we don't need the "type" argument, so I deleted it.